static/js/product_detail.js

```diff
@@ -2,54 +2,49 @@
 /*eslint semi: ["error", "always"]*/
 
 export default function productDetails() {
-  // FAQs
-  $("body").on("click", ".readmore", function(e) {
-    const $readmoreText = $(this).find(".readmore-text");
-    e.preventDefault();
-    $("li.extra-faq").toggleClass("d-none");
-    $readmoreText.text($readmoreText.text() === "Hide" ? "Read" : "Hide");
-  });
-
-  // More Dates
-  $(".dates-tooltip").popover({
-    template:
-      '<div class="popover" role="tooltip">' +
-      '<div class="arrow"></div>' +
-      '<div class="popover-header py-2 px-0 mx-5"></div>' +
-      '<div class="popover-body"></div>' +
-      "</div>"
-  });
-
   const navbar = $("#subNavBarContainer");
 
-  // Navigation Bar
-  $("body").scrollspy({
-    target: "#subNavBar",
-    offset: navbar.outerHeight() + 5.0
-  });
+  // It will make sure that the script only runs when on product page
```

Comment:
Should we make this comment a little more generic with respect to the `Navbar` instead of `product page`?

Reply:
Updated.